src/common/filesystem.c

```diff
@@ -789,15 +806,23 @@ FS_LoadPK3(const char *packPath)
 	{
 		unzClose(handle);
 		Com_Error(ERR_FATAL, "FS_LoadPK3: '%s' is not a pack file", packPath);
+		return NULL;
 	}
 
 	numFiles = global.number_entry;
 
-	if ((numFiles > MAX_FILES_IN_PACK) || (numFiles == 0))
+	if (numFiles > MAX_FILES_IN_PACK)
```

Comment:
I think in case of zip this check is superfluous - I guess having it was a bug that came from copy-pasting the .pak code and forgetting to remove if

Reply:
I have removed warning about MAX_FILES_IN_PAK for pkz.